lib/cdo/cdo_cli.rb

```diff
@@ -0,0 +1,32 @@
+#
+# Code.org command-line script utility methods
+#
+module CdoCli
+  # Terminal style utilities for fashionable output
+  def stylize(text, color_code)
+    "\e[#{color_code}m#{text}\e[0m"
+  end
+
+  def bold(text); stylize(text, 1); end
+  def dim(text); stylize(text, 2); end
+  def underline(text); stylize(text, 4); end
```

Comment:
Feel encouraged (though not required) to make these methods non-single line.
https://github.com/bbatsov/ruby-style-guide#no-single-line-methods